query-algebrizer/src/types.rs

```diff
@@ -294,6 +295,7 @@ pub enum ColumnConstraint {
         right: QueryValue,
     },
     HasType(TableAlias, ValueType),
+    NotExists(ComputedTable),
```

Comment:
Crazy GitHub bug: this line was showing as a third arm in `ComputedTable`, so there was `ComputedTable::Subquery` and `ComputedTable::NotExists`.